consult-tramp.el

```diff
@@ -44,23 +64,13 @@
   :group 'consult-tramp
   :type 'string)
 
-(defcustom consult-tramp-enable-shosts t
-  "Use known_hosts as a completion source."
-  :group 'consult-tramp
-  :type 'string)
-
 (defcustom consult-tramp-known-hosts "~/.ssh/known_hosts"
```

Comment:
I wonder about the known_hosts file. On my system this file is not readable.

Reply:
It also doesn't seem to parse anything correctly for me, I'll probably end up leaving it out of the consult-dir PR

Reply:
Maybe this code was taken from counsel-tramp/helm-tramp which is pretty dated? It is a long time ago that the format of known_hosts changed to the unreadable format.

Reply:
Likely, the comment header indicated it was copied and tweaked